src/lib/ride.ts

```diff
@@ -0,0 +1,39 @@
+import RideModel from '../database/model/ride';
+import {dataBase} from '../database';
+
+export function startRide() {
+  // TODO stop any current rides.
```

Comment:
For now the UI won't let you give you an option to start a ride while one is active. We probably want a safety net at some point but I've left it out for now. We'll have to see if it actually happens.

Reply:
I think for now this is fine 👍